.infra/index.ts

```diff
@@ -63,8 +63,8 @@ if (isAdhocEnv) {
 const redis = new Redis(`${name}-redis`, {
   isAdhocEnv,
   name: `${name}-redis`,
-  tier: 'STANDARD_HA',
-  memorySizeGb: 10,
+  tier: 'BASIC',
+  memorySizeGb: 2,
```

Comment:
```suggestion
  memorySizeGb: 3,
```
We settled at 3, cc: @idoshamun